docs/data/date-pickers/custom-field/BrowserV7Field.tsx

```diff
@@ -32,91 +32,81 @@ const BrowserFieldContent = styled('div', { name: 'BrowserField', slot: 'Content
   },
 );
 
-const BrowserDateField = React.forwardRef(
-  (props: DatePickerFieldProps, ref: React.Ref<HTMLDivElement>) => {
-    const fieldResponse = useDateField<true, typeof props>(props);
+function BrowserDateField(props: DatePickerFieldProps) {
```

Comment:
These functions moved from `forwardedRef`s to regular FCs because the refs are now provided the be context?

Reply:
Yes, the picker now passes the ref via the context.
I removed the `forwardRef` in the doc example to make clear that it is no longer needed :+1: